### PR TITLE
Added reversed relation using mapped config

### PR DESCRIPTION
**Pull request by Eighke**
It can probably be improved, but it allows to inverse a multiple relationship. This way we can add inverse relation directly from the admin or programmatically via a seeder or observer, etc.

### CODE REVIEW

File: resources/config/config.php [hunk 1]
```diff
@@ -36,4 +36,11 @@
             'min' => 1,
         ],
     ],
+    'mapped'     => [
+        'required' => true,
```
Comment (RyanThompson) [inline resources/config/config.php]: Probably should not be required.

**Comment by RyanThompson** [comment]
So the idea is that this would be assigned once to the object and assigned once as mapped to the children?

**Comment by Eighke** [comment]
> So the idea is that this would be assigned once to the object and assigned once as mapped to the children?

Exactly, we create a normal multiple.ft as usual, and we can create the mapped relation in the related model.

File: resources/config/config.php (hunk 1 above)
Comment (Eighke) [reply resources/config/config.php]: Right it is a bool.

**Comment by RyanThompson** [comment]
I think this needs to go into a callback or something.. Smells off! I like the direction though! Reverse relations are super handy.

**Pull request closed without merging**